openshift-container-platform-4/policies/SI-System_and_Information_Integrity/component.yaml

```diff
@@ -89,6 +89,25 @@
         benchmarks to remediate high, and moderate risk flaws within a
         customer defined period after a flaws discovery.
         */'
+
+- control_key: SI-2(4)
```

Comment:
```suggestion
- control_key: SI-2 (4)
```

Reply:
@mammerman Downstream tooling performs exact matches on the controls, which makes the space especially important. Easy to overlook!